app/assets/javascripts/student_profile_v2/student_profile_header.js

```diff
@@ -0,0 +1,55 @@
+(function() {
+  window.shared || (window.shared = {});
+  var dom = window.shared.ReactHelpers.dom;
+  var createEl = window.shared.ReactHelpers.createEl;
+  var merge = window.shared.ReactHelpers.merge;
+  var Routes = window.shared.Routes;
+
+  var styles = {
```

Comment:
These define styles (similar to what you would do in CSS), and you use them in the component (see https://facebook.github.io/react/tips/inline-styles.html).

This approach somewhat controversial and bleeding edge (rather than using CSS), but it's just as simple to use.  If you're curious about the rationale, [these slides](https://speakerdeck.com/vjeux/react-css-in-js) has more context about challenges with using CSS.